index.html

```diff
@@ -774,22 +710,10 @@ <h1 class="title">Our Clubs </h1>
 				<!--Row-->
 				<div class="row">
 					<div class="col-sm-12 text-center">
-						<div class="col-md-2 col-sm-2 ">
-							<div class="block-club">
-								<img src="img/osc.png" alt="">
+							<div class="col-md-4 col-sm-4 block-club">
+								<img src="img/logo/logo.png" alt="">
+								<span class="DSC-JIIT-text">DSC JIIT</span>
```

Comment:
This is not getting reflected, please add the banner of our developer society [DSC JIIT](https://dscjiit.live) and link it to our website.

Reply:
Hi, do you have access to the banner or how you want the banner to look?